packages/cli/src/cli-commands/cli-version-commands.ts

```diff
@@ -180,6 +185,15 @@ exports.builder = (yargs, composed) => {
       requiresArg: true,
       defaultDescription: 'v',
     },
+    'no-update-root-lock-file': {
```

Comment:
Is this the way how negations are handled?

Reply:
yeah that is how yargs, which is what Lerna uses for the CLI, does